config/locales/ypk.yml

```diff
@@ -0,0 +1,135 @@
+﻿ypk:
+
+# Default page title of search results page
+  default_serp_title: "%{query} - %{site_name} Ivaghillghem pillghi"
+
+# Labels for the search box
+  search: "Ivaghi"
+  search_label: "Kanaghtigu akuzita ivaghyukan"
+
+# Labels for the navigation bar
+  everything: "Qamalluku"
+  images: "Igaqellghet"
+  videos: "Muvingllat"
+  show_more: "Wata"
+
+# Labels to change the sort order by date or relevance
+  by_date: "Qavnganghhaghtat"
+  by_relevance: "Itagnallgha naalkegkam"
+
+# Labels to narrow results by a preset timeframe or a custom date range, or clear filters ('Before' is the label for custom date range with only a 'To' date set.)
+  all_time: "Alngunak qakun"
+  last_hour: "Kingulighmi awami"
+  last_day: "Kinguliq aghneq"
+  last_week: "Akuwaaq"
+  last_month: "Aawhkan"
+  last_year: "Awatangani"
+  custom_range: "Nakughpigaat tagtalet"
+  from: "Whaken"
+  to: "Taamun"
+# It's safe to modify the order of these (e.g., '%m/%d/%Y'), but do not localize the Latin letters 'd', 'm', or 'Y'. Do not change the case of the letters.
+  cdr_format: "%m/%d/%Y"
+  before: "Sivungani"
+  clear: "Ghevestigu"
+
+# Prompts for spelling, spelling overrides, and did you mean suggestions
+  showing_results_for: "Kelgutaqii naalkellghi %{corrected_query}"
+  search_instead_for: "Yaghlegi ivaghhu una %{original_query}"
+
+# Prompts for searchers to revise or refine search
+  no_results_for_and_try: "Qivghukamken, naalkusimangitukut uumun %{query}. Aatak kanaghtigu palaqaniin nalighmeng ilanganeng akuzitneng."
+  too_long: "Ivaqan aghsaan taakepigtuq. Allameng nanghiqwaa takitkughmelngughmeng akuzitmeng nalighmeng salleghmeng."
+  empty_query: "Kanaghtiighten akuzitmeng ivaghyaghqameng pikegkumi kanaghfigmi."
+
+  # Labels for inline best bets, news, and related searches modules
+  recommended: "Una pisqesugaat"
+  default_rss_govbox_label: "Ungpat"
+  related_searches: "Ilangi tawatelnguut ivaqam"
+
+# Labels for paging through results
+  page: "Qukfighneq"
+  prev_label: "Sivulighhaa"
+  next_label: "Kingulighhaa"
+
+  searches:
+    by: "uukun"
+# Prompt for searchers to skip to main content (most easily seen by tabbing through results page)
+    skip_to_main_content: "Naghughten pighpigaghmun"
+# Labels for navigation bar "sections" (most easily seen on mobile dimension)
+    menu: "Nakmiksaghqat"
+    browse_site: "Yuvghiighhu una"
+    related_sites: "Esghaghhu pikaq"
+    results_count: "%{count} naalkegkat"
```

Comment:
We are redoing this in the template soon, please make this `results_count` line into the following kind of array, but NOTE! we need to run this by your translator, because `naalkegkat` works for plurals and may also work for zero and 1, but maybe the singular and null versions are different.

```
    results_count: 
      zero: '0 naalkegkat'
      one: '1 naalkegkat'
      other: '%{formatted_count} naalkegkat'
```